WorkAndFun/src/main/java/pl/infoshare/Menu.java

```diff
@@ -0,0 +1,63 @@
+package pl.infoshare;
+
+import java.util.Scanner;
+
+public class Menu {
+    public void menu() {
+        System.out.println("----------------------------------------------------");
+        System.out.println("&&&&&&&&&&&--->> Work & Fun Service <<---&&&&&&&&&&&");
+        System.out.println("----------------------------------------------------");
+        System.out.println("Wybierz jedna z opcji wprowadzajac odpowiednia cyfre");
+        System.out.println("1 - Wyszukaj ogloszenie z oferowana usluga");
+        System.out.println("2 - Wyszukaj ogloszenie z poszukiwana usluga");
+        System.out.println("3 - Dodaj ogloszenie z oferowana usluga");
+        System.out.println("4 - Dodaj ogloszenie z poszukiwana usluga");
+        System.out.println("5 - Edytuj ogloszenie");
+        System.out.println("6 - Usun ogloszenie");
+        System.out.println("7 - Zakoncz program");
```

Comment:
```suggestion
        System.out.println("0 - Zakoncz program");
```
tak na wszelki wypadek, gdyby doszła opcja 7, 8 itd.